examples/AdWords/v201607/CampaignManagement/AddCompleteCampaignsUsingBatchJob.php

```diff
@@ -112,9 +112,10 @@ function AddCompleteCampaignUsingBatchJobExample(AdWordsUser $user) {
         $batchJob->status);
 
     $pollAttempts++;
-    if ($batchJob->status !== 'ACTIVE' &&
-        $batchJob->status !== 'AWAITING_FILE' &&
-        $batchJob->status !== 'CANCELING') {
+    if (!in_array($batchJob->status, array(
```

Comment:
Hi @shakaran thanks for your PR. We don't think using `in_array` here is necessary. You're building an array and then searching through it for each element, it adds unnecessary work. Because there are only 3 statuses to check here, we'd prefer just checking against each one - we believe this is more readable.

Thanks for pointing out that we should be using the generated constants for these enum values, we missed this when porting over examples from the old client library to this one.

As for the style, you're right we don't publish any kind of style guide. We more or less [follow our Java style guide](https://google.github.io/styleguide/javaguide.html) when things are applicable to PHP as well (e.g., indentation). Thanks for your feedback on this, we will take a look at the lint bot.

For indentation it's 2 spaces and 4 spaces for line continuations. Let us know if you have further questions. Thanks.